### ISSUE TITLE
Logging Framework?

### ISSUE THREAD

**Issue by JanKoppe** [hikhvar/mqtt2prometheus#23]
Hey,

many thanks for this cool project! Will use it in a new project, where there will be a very huge amount of telemetry reports translated into Prometheus. Right now every message will generate a single log line, as shown here: https://github.com/hikhvar/mqtt2prometheus/blob/master/pkg/metrics/ingest.go#L130

I don't suppose there is a way to configure a less verbose logging level? If not, any plans to add some sort of basic logging framework?

**Comment by hikhvar** [comment]
Hey, until now the logging was fine for me. Sure, with increasing load the current logging is sub optimal. I created a PR (#24 ) to introduce go.uber.org/zap as the logging library. I welcome feedback. The current log settings exposed to the user are the level (debug, info, warning, error, panic) with the default "info". The format can be choosen between "console":
```
2020-10-29T20:54:30+01:00	info	mqttclient/mqttClient.go:18	Connected to MQTT Broker
2020-10-29T20:54:30+01:00	info	mqttclient/mqttClient.go:19	Will subscribe to topic	{"topic": "v1/devices/me"}
```
and "json":
```
{"level":"info","ts":1604001457.4727054,"caller":"mqttclient/mqttClient.go:18","msg":"Connected to MQTT Broker"}
{"level":"info","ts":1604001457.4727683,"caller":"mqttclient/mqttClient.go:19","msg":"Will subscribe to topic","topic":"v1/devices/me"}
```

**Comment by hikhvar** [comment]
Moreover, if you are able to speak about your project, I really like to hear about the cool problems people solve with this exporter.

**Comment by hikhvar** [comment]
Implemented in Release v0.1.4.
